tests/test_session.py

```diff
@@ -185,12 +185,14 @@ async def test_creating_many_sessions_for_one_user_and_looping():
 
     for handle in session_handles:
         info = await get_session_information(handle)
```

Comment:
need to add more tests to make sure that update access token payload, update session data etc return the right boolean values based on the input.

Reply:
done.